enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/grpc/ExtAuthService.java

```diff
@@ -164,16 +164,18 @@ private CheckResponse buildResponse(CheckRequest request, ResponseObject respons
                 HeaderValueOption headerValueOption = HeaderValueOption.newBuilder()
                         .setHeader(HeaderValue.newBuilder().setKey(APIConstants.PATH_HEADER).setValue(constructedPath)
                                 .build()).build();
-                okResponseBuilder.addHeaders(headerValueOption);
+                    okResponseBuilder.addHeaders(headerValueOption);
```

Comment:
Let's revert this by removing this indentation.

Reply:
Fixed. [https://github.com/wso2/product-microgateway/pull/3278/commits/7530563cefd13bea91da379804770bc0a9c8f99f](https://github.com/wso2/product-microgateway/pull/3278/commits/7530563cefd13bea91da379804770bc0a9c8f99f)

Reply:
Shall we recheck this indentation comment again ?